pkg/apis/migration.harvesterhci.io/v1beta1/virtualmachines.go

```diff
@@ -33,6 +43,21 @@ type VirtualMachineImportSpec struct {
 	Folder       string           `json:"folder,omitempty"`
 	Mapping      []NetworkMapping `json:"networkMapping,omitempty"` //If empty new VirtualMachineImport will be mapped to Management Network
 	StorageClass string           `json:"storageClass,omitempty"`
+
+	// +optional
+	// GracefulShutdown is a flag to indicate whether the guest OS should be
+	// shutdown instead of performing a power off before the export is
+	// started. If the graceful shutdown is not done within the specified
+	// timeout, the VM will be powered off.
+	// Note, this is only supported by the VMware importer, OpenStack is
+	// performing a graceful shutdown by default.
+	// Defaults to true for the VMware importer if not disabled explicitly.
+	GracefulShutdown *bool `json:"gracefulShutdown,omitempty"`
+
+	// GracefulShutdownTimeout is the time in nanoseconds to wait for the guest
```

Comment:
can we please change the timeout to be measured in seconds?

Reply:
This makes sense, because the specification in nanoseconds is not user-friendly.

I've changed the field name and type according to this from kubevirt:
```
TimeoutSeconds int32 `json:"timeoutSeconds,omitempty"`
```